google_guest_agent/command/command_monitor.go

```diff
@@ -226,3 +230,36 @@ func (c *Server) start(ctx context.Context) error {
 	c.srv = srv
 	return nil
 }
+
+type getOptionRequest struct {
+	Request
+	Option string
+}
+
+type getOptionResponse struct {
+	Response
+	Option string
+	Value  string
+}
+
+func getOption(b []byte) ([]byte, error) {
+	var req getOptionRequest
+	json.Unmarshal(b, &req)
+	var resp getOptionResponse
+	resp.Option = req.Option
+	if req.Option == "" {
+		resp.Status = 2
+		resp.StatusMessage = "No option specified"
+		return json.Marshal(resp)
+	}
+	var opt any
+	opt = cfg.Get()
+	for _, k := range strings.Split(resp.Option, ".") {
```

Comment:
is this a generic config listing command?

Reply:
Yep. Documentation should hopefully make this clearer now. It only lives here because I couldn't think of a better place to put it, and if it goes in `cfg` it creates a circular dependency.